### DISCUSSION TITLE
Allow Field Selection by Field Group from Fluid Fields

### DISCUSSION THREAD

**Discussion by acalvino4** [ExpressionEngine/ExpressionEngine#2890] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
When creating a fluid field, I have to select the allowed fields individually, like below:
![image](https://user-images.githubusercontent.com/43254861/137372471-66e67820-0943-4fed-a9f5-a8ef20986f33.png)

**Describe the solution you'd like**
It would be very convenient if I could select by Field Group, like is possible when creating a channel:
![image](https://user-images.githubusercontent.com/43254861/137372602-da233d26-2b05-4bd7-a72f-d512e6b074e6.png)

**Describe alternatives you've considered**
The alternative is the status quo.

**Teachability, Documentation, Adoption, Migration Strategy**
If the interface was the same as field selection for channels, everyone would already be familiar with it. I am sure a backwards-compatible solution is possible.

**Comment by intoeetive** [comment]
[Field Groups in Fluid](https://docs.expressionengine.com/latest/fieldtypes/fluid.html#custom-field-groups) are now supported in ExpressionEngine
